.env.example

```
@@ -0,0 +1,102 @@
+S3G_INTERNAL_PORT=9878
+SCM_UI_PORT=9870
+S3G_PORT=9878
+
+# Spark Configuration
+SPARK_DRIVER_MEMORY=4g
+SPARK_EXECUTOR_MEMORY=4g
+SPARK_DRIVER_MAXRESULTSIZE=2g
+
+# JupyterHub Configuration
+JUPYTERHUB_PORT=8000
+JUPYTERHUB_PASSWORD=test123
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JUPYTERHUB_PASSWORD key should go before the JUPYTERHUB_PORT key

(UnorderedKey)